ADAL/src/public/ADErrorCodes.h

```diff
@@ -93,6 +93,9 @@ typedef NS_ENUM(NSInteger, ADErrorCode)
     
     /*! Invalid data was returned from the server, see -errorDetails for more information. */
     AD_ERROR_SERVER_INVALID_RESPONSE = 212,
+
+    /*! Intune App Protection is required by the service */
```

Comment:
@jasoncoolmax Can we add some more info for the integrating app of what's expected behavior for this error?

Reply:
What about "The requested resource is protected by an Intune Conditional Access policy.  The calling app should integrate the Intune SDK, please see https://aka.ms/intuneMAMSDK for more information."